src/components/ServerInputErrors.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div class="container" v-if="cleanedMessages">
+    Address the following:
+    <b-list-group>
+      <b-list-group-item v-for="message in cleanedMessages" variant="danger">
+        {{ message }}
+      </b-list-group-item>
+    </b-list-group>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "ServerInputErrors",
+  data() {
+    return {
+      errorListKey: 0
+    };
+  },
+  props: ["graphError"],
+  computed: {
+    cleanedMessages: function() {
```

Comment:
Read the catched error and loop over all the errors and clean up the text.
This is a computed variable. We use this for displaying the errors